### PR TITLE
ON-19952 # fixed elements under a section in repeatable sets not bein…

### PR DESCRIPTION
**Pull request by aaronroworth**
…g pre-filled

### CODE REVIEW

**Comment by divporter** [comment]
LGTM

File: src/services/generate-default-data.ts [hunk 1]
```diff
@@ -200,7 +200,11 @@ function parseFormSubmissionModel(
   return parseUnknownAsRecord(value, (record) => {
     return elements.reduce<FormSubmissionModel>((model, element) => {
       switch (element.type) {
-        case 'section':
+        case 'section': {
+          const partialModel = parseFormSubmissionModel(element.elements, model)
+          Object.assign(model, partialModel)
+          break
+        }
```
Comment (mymattcarroll) [inline src/services/generate-default-data.ts:207]:
```suggestion
        case 'section':
```

Comment (aaronroworth) [reply src/services/generate-default-data.ts:207]: that would just change it back to what it was originally?

Comment (mymattcarroll) [reply src/services/generate-default-data.ts:207]: ah, lol. 

If sections are broken, wouldn't the same bug be there for pages as well?

Can you please confirm the same bug does not exist for pages?

**Pull request merged**